packages/projects-docs/pages/learn/repositories/overview.mdx

```diff
@@ -27,3 +27,16 @@ When you press `Create Branch`, we automatically create a new branch and set up
 <Video src="../../repo-branching.mp4" />
 
 If you are looking at someone’s PR and want to quickly test a suggestion, simply click `Create Branch`, write the code and share a link to your changes. If you’re happy with the changes, merge that into the existing PR. All of this in under a minute.
+
+## Understanding repository imports
+
+CodeSandbox handles imported repositories on a per-workspace basis. As a result, any given repository may exist in different states if it has been imported to multiple CodeSandbox workspaces. Additionally, the public version of a repository will always have its own, independent state.
```

Comment:
I think maybe a confusion here is stating "CodeSandbox" workspace and the image shows an actual CodeSandbox workspace... so might be confusion about general CodeSandbox workspaces and the image 😄